web/src/app/schedules/temp-sched/TempSchedAddShiftsStep.tsx

```diff
@@ -211,20 +232,44 @@ export default function TempSchedAddShiftsStep({
         </Grid>
 
         {/* shifts list container */}
-        <Grid item xs={6} className={classes.listOuterContainer}>
-          <div className={classes.listInnerContainer}>
-            <TempSchedShiftsList
-              scheduleID={scheduleID}
-              value={value}
-              start={start}
-              end={end}
-              onRemove={(shift: Shift) => {
-                setShift(shift)
-                onChange(value.filter((s) => !shiftEquals(shift, s)))
-              }}
-              edit={edit}
-            />
+        <Grid item xs={6} className={classes.shiftsListContainer}>
+          <div className={classes.listOuterContainer}>
+            <div className={classes.listInnerContainer}>
+              <TempSchedShiftsList
+                scheduleID={scheduleID}
+                value={value}
+                start={start}
+                end={end}
+                onRemove={(shift: Shift) => {
+                  setShift(shift)
+                  onChange(value.filter((s) => !shiftEquals(shift, s)))
+                }}
+                edit={edit}
+              />
+            </div>
           </div>
+          {showCoverageGapsWarning && (
+            <Alert severity='error' className={classes.noCoverageError}>
+              <AlertTitle>Gaps in coverage</AlertTitle>
+              <FormHelperText>
+                There are gaps in coverage. During these gaps nobody will
+                receive alerts. If you still want to proceed, check the box and
+                then click Retry.
```

Comment:
user could be making actions with something other than a mouse

```suggestion
                then retry.
```

Reply:
True, we could also add in the sched name to the second sentence:

```
There are gaps in coverage. During these gaps, nobody on <schedule_name> will
receive incoming alerts. If you still want to proceed, check the box and retry.
```

Reply:
I just put "this schedule" for the schedule name. If I wanted to fetch the schedule name I could use a context from the scheduleDetails or make another query in this component to get the name from the sched id. I'm a little weary to have the dialog reliant on a parent context though since it's a dialog. I'd be interested to hear your thoughts on this.

Reply:
I think "this schedule" works well. It puts more emphasis on the page they're on, where-as I could see them second guessing themselves for a moment if we put the full schedule name in, thinking "am I configuring the right thing?".